src/MSIdentityScaffolding/README.md

```diff
@@ -34,44 +38,64 @@ Usage:
   dotnet-msidentity [command] [options]
```

Comment:
Should this be `dotnet msidentity`?

Reply:
both  "dotnet msidentity" and "dotnet-msidentity" work but "dotnet msidentity" is more consistent with our other tools.